deploy/controller/crds.yaml

```diff
@@ -164,6 +164,13 @@ spec:
               driver:
                 description: Driver contains  the name of CSI driver.
                 type: string
+              fenceState:
```

Comment:
Should this be included in the `additionalPrinterColumns`?

I guess it also is required to update the version?

Reply:
ACK to "additionalPrinterColumns"

Question about the version. Since the version is still v1alpha1 for the api and ramen being the only consumer of NetworkFence resource as of now, do you thing it is necessary to update the version?

Reply:
I don't know... That is probably something @Yuggupta27 can say a little more about.

Reply:
Updating the version depends on if we expect more major changes to the CRD. If we expect more such changes in the recent future, better to stay with `v1alpha1` for now. What do you think?